app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsView.kt

```diff
@@ -165,6 +170,7 @@ interface PaymentMethodsView {
     SANDBOX("sandbox"),
     ASK_FRIEND("ask_friend"),
     CHALLENGE_REWARD("challenge_reward"),
-    VKPAY("vk_pay")
+    VKPAY("vk_pay"),
+    GOOGLEPAY_WEB("googlepay"),
```

Comment:
Shouldn’t we use only one constant instead of define the name many times?

Reply:
They have different purposes. Analytics vs payment key. They are not defined the same in all other payments